src/sets.jl

```diff
@@ -287,6 +287,34 @@ function Base.:(==)(set1::S, set2::S) where S <: Union{PowerCone, DualPowerCone}
     return set1.exponent == set2.exponent
 end
 
+"""
+    NormSpectralCone{T <: Real}(row_dim, column_dim)
+
+The epigraph of the matrix spectral norm (maximum singular value function) ``\\{ (t, X) \\in \\mathbb{R}^{1 + row_dim \\times column_dim} : t \\ge \\sigma_1(X) \\}`` where ``\\sigma_i`` is the ``i``th singular value of the general matrix ``X`` of row dimension `row_dim` and column dimension `column_dim`.
```

Comment:
What is a "general matrix" ?
How are the singular values ordered ?

Reply:
I'm trying to distinguish from the various symmetric matrix sets already there. Happy to just say "matrix" instead of "general matrix".